### PR TITLE
#17400 concepts docs

### PR DESCRIPTION
**Pull request by bbefort**
DRAFT of the Concepts page, following the outline I found [here](https://diveplane.visualstudio.com/dp/_wiki/wikis/dp.wiki/758/Howso-Engine-Documentation-Sitemap). These still need editing, but would like others' thoughts as I go forward to make sure I'm on the right track.

### CODE REVIEW

File: source/user_guide/concepts.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Conviction
+----------
+
+The connection between IBL and information theory becomes apparent in the concept of **conviction**, which is a measure of surprisal. Conviction is the computed ratio of actual information to expected information, with
+a range of zero to infinity. A conviction value of one is average, as the actual information is equal to the expected information. Convictions less than one indicate higher surprisal, as the actual information
+iss less than the expected information, while convictions greater than one indicate lower surprisal, since the actual information is higher than the expected information. 
+
+Actual information is the amount of information that a data point actually adds when it is evaluated. Expected information is what the value of a new data point should be on average, given
+all knowledge of the data around it. For example, say you are again trying to make heart disease predictions, given historical heart disease data and a new patient's data is being analyzed.
```
Comment (jackx111) [inline source/user_guide/concepts.rst:84]: given historical heart disease data,

File: source/user_guide/concepts.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+This patient has features very similar to historical data with positive prediction status. Thus, its expected heart disease prediction value is positive. Its actual information, though, is the true information
```
Comment (jackx111) [inline source/user_guide/concepts.rst:85]: minor suggestion. referring to a patient as it sounds a bit weird. Alternatives - the case('s), the patient('s). Up to you

File: source/user_guide/concepts.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    Howso will be confident in its decision of a positive prediction. In contrast, you might have a patient who is extremely young and fit, but with high cholesterol levels, whose information doesn't match the information
+    of many other patients. Even if Howso predicts a positive diagnosis for this young patient, because this patient's information is in such a sparse region of dataspace, Howso will tell you it has low confidence
+    in its prediction, indicating further investigation may be necessary.
+
+Conviction
+----------
+
+The connection between IBL and information theory becomes apparent in the concept of **conviction**, which is a measure of surprisal. Conviction is the computed ratio of actual information to expected information, with
+a range of zero to infinity. A conviction value of one is average, as the actual information is equal to the expected information. Convictions less than one indicate higher surprisal, as the actual information
+iss less than the expected information, while convictions greater than one indicate lower surprisal, since the actual information is higher than the expected information.
```
Comment (jackx111) [inline source/user_guide/concepts.rst:81]: might want a very brief description of surprisal

File: source/user_guide/concepts.rst [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+Once a `Trainee` is built, i.e., the data is stored, it is `analyzed` to understand the relationships between the individual data points. Here, the internal parameters describing these
+relationships are tuned to improve performance and the accuracy of predictions and metrics. 
+
+React
+-----
+
+Finally, after building the `Trainee` and `analyzing`, Howso Engine can be used for a variety of data-driven analysis applications. Often, these applications involve
+`reacting` to new data, which involves exposing the Trainee to new information and using the underlying IBL algorithm to make a prediction about that new information.
+This is equivalent to predicting or labeling in many traditional machine learning workflows, although `reacting` can be used for a variety of analyses, in addition to supervised
```
Comment (jackx111) [inline source/user_guide/concepts.rst:112]: React into trainee is a important part of workflow too, you don't need to specifically call it out but I would say something along the lines of "React can also be used to analyze data already trained into the Trainee etc..".

**Review by sdconrox — APPROVED** [review]

**Pull request merged**